### DISCUSSION TITLE
Lost access to an important diagram in Google Drive, unable to recover with shared link

### DISCUSSION THREAD

**Discussion by nicosoca** [jgraph/drawio#4739] [General]
Hello,

I’m reaching out because I lost access to an important diagram that I created and stored using Google Drive. I still have the link to the file, but when I try to open it, I receive a message that the file does not exist. I cannot locate it directly in Google Drive either.

Is there any way to recover this file or access a backup using the link? Any guidance or potential solutions would be greatly appreciated.

What I have is the shared link; I'm not sure if I was the owner of the diagram. I’ve already checked my Google Drive account and didn’t find anything in the trash.

Thank you for your help!

**Comment by davidjgraph** [comment]
We store nothing, it's only in your Google Drive. If you're not the owner then, yes, you don't have control over whether the file is deleted.

**Comment by nicosoca** [comment]
I understand they don't store it. I also contacted Google Drive. Could you
tell me a link or some type of identifier for the Google Drive file where
it was persisted so I can send it to the Google Drive support team?

**Comment by davidjgraph** [comment]
We store nothing, not even IDs. We have no access to anything.

**Comment by nicosoca** [comment]
Thanks
